Services/Sources/Services/Auth/Components/AuthenticationError.swift

```diff
@@ -1,10 +1,10 @@
-import Foundation
+import Amplify
 
 public typealias AuthenticationErrorDescription = String
 public typealias AuthenticationRecoverySuggestion = String
 public typealias AuthenticationField = String
 
-public enum AuthenticationError {
+public enum AuthenticationError: AmplifyError {
```

Comment:
From my point of view, that's not a good solution to use specific library error when we had created our special error in order to hide realisation. Let's discuss.

Reply:
Good point, @diantyn. I have just reconsidered my code.